web/client/utils/CoordinatesUtils.js

```diff
@@ -207,6 +207,22 @@ export const  reprojectExtent = (extent, projection, isIDL) => {
         ].reduce((a, b) => [...a, b.x, b.y], []));
 };
 
+/**
+ * Get extent by EPSG projection
+ * @param projection {string} the projection of the needed extent
+ * @return {array} extent [minx, miny, maxy, maxy]
+ */
+export const  getExtentByCRSCode = (projection) => {
```

Comment:
The requirement ask for entered coordinates are within the CRS bounds. i.e for `EPSG:2154` the bounds are approximately [-9.86, 41.15, 10.38, 51.56]. So the value should be limited in these ranges

Reply:
@mahmoudadel54 my question is: why such requirement is so ambiguous for you? 
If one of the coordinates entered by the user is outside the CRS bounds the validator should notify the error. I don't know how to explain it better.